### DISCUSSION TITLE
GH Actions and limits

### DISCUSSION THREAD

**Discussion by dtiziani** [community/community#62753] [Actions] [answered]
### Select Topic Area

Question

### Body

If I use hosted runners in my AWS, does any of the GitHub limits and pricing applies? I imagine not, but could not find a reference to it on the documentation. thanks!

**Comment by elderguardian** [comment, marked as answer]
Hello, 

If you're using your own runners from AWS with GitHub Actions, the rules and costs mentioned in GitHub's documentation don't apply. The rules about how much you can use and how much you pay are for GitHub's runners. If you're using runners from AWS, you should look at AWS's information page.

**Reply by dtiziani** [reply]
thanks, just wanted to make sure!
